src/lib/customPropTypes.js

```diff
@@ -181,6 +181,49 @@ export const demand = (requiredProps) => {
 }
 
 /**
+ * Ensure a component can render as a node passed as a prop value in place of children.
+ */
+export const content = (...args) => every([
```

Comment:
Apologies for the lack of response when you asked about this previously, but I really liked having the longer names here.  I think it is more clear and more in line with the React community to be explicit.  Especially being `collection` is a legitimate data type and this refers to something different, shorthand.

```jsx
contentShorthand
itemShorthand
collectionShorthand
```

Given all the back and forth you suffered here, if you wanna push this off to another PR, I get that and am OK doing so. :)